bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -277,6 +277,27 @@
 		<config-description-ref uri="thing-type:avmfritz:fritzdevice"/>
 	</thing-type>
 
+	<thing-type id="HAN_FUN_BLIND">
```

Comment:
I am wondering if this should be `HAN_FUN_BLINDS` - be aware of the final "S" as blinds is the correct translation.

Reply:
As AVM uses the term blind and, according to dict.leo.org, Rolladen can be translated as roller blind, I would prefer to stick to HAN_FUN_BLIND.

Reply:
Hi, I'm not shire if I can explain it correctly in English.
My experience from my former work: if we talk about the complete device we call it blinds.
If we talk about an ui item to control something in this device, we talk about a blind control. 
Maybe that helps

Reply:
I've followed your advice and have changed the term blind to blinds.